### PR TITLE
[Task] Updates Readme

### PR DESCRIPTION
**Pull request by hvilloria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+```
+```
+  docker-compose up
+```
 
-- React en http://localhost:3000
-- Phoenix en http://localhost:4000
-- Postgres en http://localhost:2345
+
+# Create Database & Run migrations
```
Comment (bian2510) [inline README.md:20]: maybe you can add here run docker-compose run --rm api mix deps.get

**Review by bian2510 — COMMENTED** [review]
JAJAJAJAJJA

File: README.md (hunk 1 above)
Comment (hvilloria) [reply README.md:20]: lets put it in getting started to assure everyone runs that command when setting up the proj.

Comment (bian2510) [reply README.md:20]: good job Alfredo

**Pull request merged**